interfaces/repo.go

```diff
@@ -817,7 +817,7 @@ func (r *Repository) UpdateHotplugSlotAttrs(ifaceName, hotplugKey string, static
 	}
 
 	for _, slotInfo := range r.slots[snapName] {
-		if slotInfo.Interface == ifaceName && slotInfo.HotplugKey == hotplugKey {
+		if slotInfo.Interface == ifaceName && (slotInfo.HotplugKey == hotplugKey) {
```

Comment:
Is this change relevant?

Reply:
Ah, not really, restored, thanks for spotting!